### PR TITLE
Added copper pot to all recipes where pot is required

### PR DESCRIPTION
**Pull request by Night-Pryanik**

### CODE REVIEW

**Comment by Chezzo** [comment]
Not using copper for a still makes whatever you are distilling taste pretty bad.

**Comment by Night-Pryanik** [comment]
> Not using copper for a still makes whatever you are distilling taste pretty bad.

Why is that?

**Comment by Chezzo** [comment]
Well, the sulfur from the yeast hooks on to the copper, making hydrogen-sulfide which forms copper sulfate. The copper sulfate sticks to the inside of the still, and then you wash it off when you are done.

**Comment by Night-Pryanik** [comment]
Okay, removed pot in still recipe.

**Comment by angoddu** [comment]
Can we make the recipe for copper pot not require tin? Tin is so uncommon that unless you find a copper pot then you can't make a still. Usually it isn't a problem as they are rather common but it makes it hard when playing on a low spawn, small city world

**Comment by kevingranade** [comment]
> Can we make the recipe for copper pot not require tin?

1. Please don't post requests in unrelated issues, the copper pot recipe isn't being touched here, so it's unrelated.
2. "It's hard to make X in Y situation" is not a reason to change a recipe.

**Comment by angoddu** [comment]
My bad. Sorry

File: data/json/recipes/recipe_electronics.json [hunk 1]
```diff
@@ -673,6 +674,7 @@
       [ "scrap", 6 ],
       [ "pan", 2 ],
       [ "pot", 2 ],
+      [ "pot_copper", 2 ],
       [ "knife_butcher", 6 ],
       [ "knife_steak", 18 ],
       [ "knife_butter", 18 ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_electronics.json]: I just noticed how idiotic this recipe is. 18 steak knives? Two whole pots (each larger than the complete maker)?

**Comment by Night-Pryanik** [comment]
Honestly I can't imagine how knives can help in crafting coffeemaker, so I lowered components reqs as in hotplate recipe.

**Comment by Coolthulhu** [comment]
I think someone wanted to allow knives as scrap metal. Can't they be uncrafted into scrap anyway? I think the knives could be just removed from the recipe.

**Comment by Xelat84** [comment]
> Well, the sulfur from the yeast hooks on to the copper, making hydrogen-sulfide which forms copper sulfate. The copper sulfate sticks to the inside of the still, and then you wash it off when you are done.

Actually this happens only when making wine, and only if grapes was sprayed with sulfur to prevent powdery mildew. There is no other ways that sulfur gets into wort. So I vote to get back regular pot into still recipe.

**Pull request merged**